src/main/java/edu/ucsb/cs156/happiercows/jobs/UpdateCowHealthJobInd.java

```diff
@@ -0,0 +1,94 @@
+package edu.ucsb.cs156.happiercows.jobs;
+
+import java.util.Optional;
+
+import edu.ucsb.cs156.happiercows.entities.Commons;
+import edu.ucsb.cs156.happiercows.entities.User;
+import edu.ucsb.cs156.happiercows.entities.UserCommons;
+import edu.ucsb.cs156.happiercows.repositories.CommonsRepository;
+import edu.ucsb.cs156.happiercows.repositories.UserCommonsRepository;
+import edu.ucsb.cs156.happiercows.repositories.UserRepository;
+import edu.ucsb.cs156.happiercows.services.jobs.JobContext;
+import edu.ucsb.cs156.happiercows.services.jobs.JobContextConsumer;
+import edu.ucsb.cs156.happiercows.strategies.CowHealthUpdateStrategy;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+public class UpdateCowHealthJobInd implements JobContextConsumer {
+
+    @Getter
+    private CommonsRepository commonsRepository;
+    @Getter
+    private UserCommonsRepository userCommonsRepository;
+    @Getter
+    private UserRepository userRepository;
+    @Getter
+    private Long commonsID;
+
+    @Override
+    public void accept(JobContext ctx) throws Exception {
+        ctx.log("Updating cow health...");
+
+       Optional<Commons> commonUpdatedOpt = commonsRepository.findById(commonsID);
+
+        if(commonUpdatedOpt.isPresent()){
+            Commons commons = commonUpdatedOpt.get();
+            ctx.log("Commons " + commons.getName() + ", degradationRate: " + commons.getDegradationRate() + ", carryingCapacity: " + commons.getCarryingCapacity());
```

Comment:
See changes to `UpdateCowHealthJob` on the line parallel to this one in PR #35 plus comments below.